### PR TITLE
Support implicit auth using environment variable

### PR DESCRIPTION
**Pull request by mattt**
Resolves https://github.com/replicate/replicate-javascript/issues/121

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,7 @@ Create the client:
 import Replicate from "replicate";
 
 const replicate = new Replicate({
-  // get your token from https://replicate.com/account
-  auth: process.env.REPLICATE_API_TOKEN,
+  auth: "my api token", // defaults to process.env.REPLICATE_API_TOKEN
```
Comment (zeke) [inline README.md:29]: I see you dropped the comment about finding the token on the site. I'm guessing that was intentional. Do you think it's easy enough for most people to find their token on the site, and this comment is just getting in the way?

**Review by zeke — APPROVED** [review]
I like it. :)

File: README.md (hunk 1 above)
Comment (mattt) [reply README.md:29]: Not at all intentional! Thanks for calling this out 😄

Comment (mattt) [reply README.md:29]:
```suggestion
  // get your token from https://replicate.com/account
  auth: "my api token", // defaults to process.env.REPLICATE_API_TOKEN
```

**Pull request merged**